GLTFSerialization/GLTFSerialization/GLTFParser.cs

```diff
@@ -19,7 +19,7 @@ internal struct GLBHeader
 			public uint FileLength { get; set; }
 		}
 		
-		public static GLTFRoot ParseJson(Stream stream, long startPosition = 0)
+		public static void ParseJson(Stream stream, ref GLTFRoot gltfRoot, long startPosition = 0)
```

Comment:
Should probably be out instead of ref

Reply:
I was starting to fix it but then noticed you had done it. Thanks!